agent/handlers/v4/response.go

```diff
@@ -121,6 +124,26 @@ func toV4NetworkResponse(
 	return resp, nil
 }
 
+// augmentContainerResponse augments the container response with additional fields.
+func augmentContainerResponse(
+	containerID string,
+	state dockerstate.TaskEngineState,
+	v4Response *tmdsv4.ContainerResponse,
+) *tmdsv4.ContainerResponse {
+	dockerContainer, ok := state.ContainerByID(containerID)
+	if !ok {
+		// did not find container, continue on and try next container(s)
+		// we don't return error here to avoid disrupting all of a TMDS response
+		// on a single missing container.
```

Comment:
Should we add a log at warning or debug level if the container cannot be found from the state?
Like https://github.com/aws/amazon-ecs-agent/blob/master/agent/handlers/v4/stats_response.go#L41-L46.

Reply:
yes I can add that

Reply:
Thanks!